chapter3/tests/t11/nginx.conf

```diff
@@ -0,0 +1,37 @@
+user nginx nginx;
+pid /run/nginx.pid;
+daemon off;
+
+events {
+}
+
+http {
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  log_format main '$remote_addr - $remote_user [$time_local] "$request" '
```

Comment:
ugly formatting :)

Reply:
Updated my vim config - hope will not repeat it.